Source/WebCore/platform/graphics/cocoa/WebMAudioUtilitiesCocoa.mm

```diff
@@ -254,7 +254,7 @@ bool parseOpusTOCData(const SharedBuffer& frameData, OpusCookieContents& cookie)
 #endif
 }
 
-bool parseOpusPrivateData(std::span<const uint8_t> codecPrivateData, SharedBuffer& frameData, OpusCookieContents& cookie)
+bool parseOpusPrivateData(std::span<const uint8_t> codecPrivateData, std::span<const uint8_t> frameData, OpusCookieContents& cookie)
```

Comment:
Pre-exisiting issue, but it is unclear whether only part of cookie values will be updated here.
We can probably just send an std::optional< OpusCookieContents> instead.

Reply:
OpusCookieContents is an out parameter, I don't see why you would want to make this an optional.

Reply:
I was thinking of:
```suggestion
std::optional<OpusCookieContents > parseOpusPrivateData(std::span<const uint8_t> codecPrivateData, std::span<const uint8_t> frameData)
```

Reply:
I see.
good idea